.pre-commit-config.yaml

```diff
@@ -56,7 +56,7 @@ repos:
       - id: script-must-have-extension
         stages: [commit]
   - repo: https://github.com/golangci/golangci-lint
-    rev: v1.43.0
+    rev: v1.61.0
```

Comment:
it was failing earlier so upgraded -
```
golangci-lint............................................................Failed
- hook id: golangci-lint
- exit code: 2

panic: load embedded ruleguard rules: rules/rules.go:13: can't load fmt

goroutine 1 [running]:
github.com/go-critic/go-critic/checkers.init.9()
        /Users/manoj.surudwad/.cache/pre-commit/repofcru5w2o/golangenv-default/pkg/mod/github.com/go-critic/go-critic@v0.6.1/checkers/checkers.go:58 +0x424
```

Reply:
Can you pull this out in a separate PR please, we'll need to backport this

Reply:
oh, will do and rebase too.

Reply:
done in 1cfdfac2ae78a972651a758ff298519dc6fff60a